docs/guides/installation.rst

```diff
@@ -1,36 +1,56 @@
 Installation
 ============
 
-This section describes how to install `HTSinfer` using Conda.
+This section describes how to install `HTSinfer`.
 
-Clone the Repository and Install Dependencies
----------------------------------------------
+Install using Conda
```

Comment:
The GitHub installation also uses Conda, therefore perhaps it's best (and more consistent wrt to the GitHub installation) to write: Install from Bioconda